test/__snapshots__/StatsTestCases.basictest.js.snap

```diff
@@ -193,6 +193,13 @@ exports[`StatsTestCases should print correct stats for async-commons-chunk-auto
     > ./a ./index.js 1:0-47
     dependent modules 60 bytes [dependent] 3 modules
     ./a.js + 1 modules 185 bytes [built] [code generated]
+
+  LOG from webpack.FileSystemInfo
+  <w> Managed item Xdir/async-commons-chunk-auto/node_modules/z.js isn't a directory or doesn't contain a package.json
```

Comment:
I'll change to correct one

Reply:
Maybe move the file to `node_modules/z/index.js` and add a single `package.json` with `{ "name": "z", version: "1.0.0" }`.